recipe/meta.yaml

```diff
@@ -66,163 +86,173 @@ outputs:
         - darts.utils
         - darts.utils.data
         - darts.utils.historical_forecasts
-      source_files:
-        - darts
-      commands:
-        - pytest darts/tests
       requires:
         - pip
         - testfixtures
         - pytest
-
-  - name: u8darts-notorch
-    build:
-      noarch: python
-    requirements:
-      host:
-        - python >=3.8
-      run:
-        - python >=3.8
-        - {{ pin_subpackage("u8darts", exact=True) }}
-        - catboost >=1.0.6
-        - lightgbm >=3.2.0
-        - prophet >=1.1.1
-
-    test:
-      imports:
-        - darts
-        - darts.ad
-        - darts.ad.aggregators
-        - darts.ad.anomaly_model
-        - darts.ad.detectors
-        - darts.ad.scorers
-        - darts.dataprocessing
-        - darts.dataprocessing.dtw
-        - darts.dataprocessing.encoders
-        - darts.dataprocessing.transformers
-        - darts.datasets
-        - darts.explainability
-        - darts.metrics
-        - darts.models
-        - darts.tests
-        - darts.tests.dataprocessing
-        - darts.tests.dataprocessing.transformers
-        - darts.tests.utils
-        - darts.utils
-        - darts.utils.data
-        - darts.utils.historical_forecasts
       source_files:
         - darts
       commands:
-        - pytest darts/tests
-      requires:
-        - pip
-        - testfixtures
-        - pytest
+        - pytest -vvv -s darts/tests
 
-  - name: u8darts-torch
-    build:
-      noarch: python
+  - name: u8darts
+    script: base_build.sh  # [unix]
+    script: base_build.bat # [win]
```

Comment:
```suggestion
```
You don't need these since you have `{{ pin_subpackage("darts", exact=True) }}` in the runtime dependencies.

Reply:
ok removed